### ISSUE TITLE
Ability to log or view issues with importing options

### ISSUE THREAD

**Issue by NicholasKatsambiris** [richtabor/MerlinWP#27]
Not sure if i have missed a setting within merlin. But when testing merlinwp in my theme the options import goes red and i'm unable to see what could be causing the issue, is there a way to log when the import fails?

**Comment by richtabor** [comment]
Hey Nicholas,

There’s not currently, though that would be nice for sure. Do you have the customizer.dat file inside the /demo/ folder? And, does it contain the right output from the Customizer Import/Export plugin? And do you have the correct demo_directory location set within your merlin-config.php file? Those are the typical reasons the options aren’t loading.

**Comment by NicholasKatsambiris** [comment]
Hey Rich, 

From what i can see, all my options are importing correctly and everything transfers across except for the nav menu. Have you ran into this issue?

**Comment by richtabor** [comment]
Yea I have. I have it importing menu locations based on get_theme_mod( 'nav_menu_locations' ) though that does not cover the actual pages/links within the menus. The issue I see there is trying to determine where the wizard's importing functionality stops. Should it determine the menu items too, or is that something that should be left to the user to do?

**Comment by NicholasKatsambiris** [comment]
If the menu location is unique and won't override any existing menu's i think it could work, especially for fresh installations and users unfamiliar with WordPress. That way once the merlin wizard has finished the website would look exactly like the theme demo.

**Comment by primozcigler** [comment]
@capuderg is the new OCDI importer writing the log file? If yes, it would be awesome to mention that in the docs (for easier debugging).

**Comment by capuderg** [comment]
@primozcigler no, there is no logging in WP Merlin. I have it on the todo list (we'll probably use monolog or something like that)
